### DISCUSSION TITLE
Fully Dedicated Mobile App

### DISCUSSION THREAD

**Discussion by SirOMGitsYOU** [pelican-dev/panel#25] [Ideas]
Similar to what was available for Ptero, Bringing back the mobile app would be great for managing the server on the go, Given the current pre-release screenshots the UI looks to be something more suitable to be on a mobile platform as well. 

A dedicated mobile app and not just using the mobile browser would be so much better as from my experience the browser-based apps (I.e Bookmarked apps on iOS) never really work all too well and usually end up having oversight or flaws with the UI design (_Happy to be proven wrong if it can be pulled off well!_)

**Comment by TekExplorer** [comment]
Thats me! featherdactyl already exists and i will be involved with the team as well to ensure its compatible

**Comment by lancepioch** [comment]
Please follow @TekExplorer for his updates, thanks!
